frontend/ui/data/koptoptions.lua

```diff
@@ -76,6 +77,136 @@ In 'semi-auto' and 'manual' modes, you may need to define areas once on an odd p
             },
         }
     },
+    {
+        icon = "resources/icons/appbar.magnify.zoom.png",
+        options = {
+            {
+                name = "zoom_overlap_h",
+                name_text = "Horizontal overlap",
+                buttonprogress = true,
+                fine_tune = true,
+                values = {0, 12, 24, 36, 48, 60, 72, 84},
+                default_pos = 4,
+                default_value = 36,
+                show_func = function(config)
+                    return config and config.zoom_mode_genus < 3
+                end,
+                event = "DefineZoomMode",
+                args =   {0, 12, 24, 36, 48, 60, 72, 84},
+                labels = {0, 12, 24, 36, 48, 60, 72, 84},
+                name_text_hold_callback = optionsutil.showValues,
+                help_text = _([[Set horizontal zoom overlap (between columns).]]),
+            },
+            {
+                name = "zoom_overlap_v",
+                name_text = "Vertical overlap",
+                buttonprogress = true,
+                fine_tune = true,
+                values = {0, 12, 24, 36, 48, 60, 72, 84},
+                default_pos = 4,
+                default_value = 36,
+                show_func = function(config)
+                    return config and config.zoom_mode_genus < 3
+                end,
+                event = "DefineZoomMode",
+                args =   {0, 12, 24, 36, 48, 60, 72, 84},
+                labels = {0, 12, 24, 36, 48, 60, 72, 84},
+                help_text = _([[Set vertical zoom overlap (between lines).]]),
```

Comment:
```suggestion
                labels = {0, 12, 24, 36, 48, 60, 72, 84},
                name_text_hold_callback = optionsutil.showValues,
                args =   {0, 12, 24, 36, 48, 60, 72, 84},
```

Reply:
When playing with this, I saw these values 12, 36... What's the unit? screen pixels, scaled-to-dpi pixels? It didn't talk to me :/

Reply:
I just wanted to add a `name_text_hold_callback` to this one also, no clue about the units

Reply:
> When playing with this, I saw these values 12, 36... What's the unit? screen pixels, scaled-to-dpi pixels? It didn't talk to me :/

They are percent (%).